utils/install_latest.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -e
+if ! command -v jq > /dev/null || ! command -v grafana-cli; then
+  echo "This script can not run without jq or grafana-cli"
+fi
+DOWNLOAD_URL=$(wget -O- -q https://api.github.com/repos/tribe29/grafana-checkmk-datasource/releases/latest | jq -r ".assets[0].browser_download_url")
+grafana-cli --pluginUrl "$DOWNLOAD_URL" plugins install tribe-29-checkmk-datasource
```

Comment:
directly calling grafana-cli is not a good idea: https://github.com/grafana/grafana/issues/25367
should check if the current user is grafana and if not, print some instructions to be called via sudo as described here: https://github.com/taosdata/grafanaplugin